virtcontainers/utils/utils.go

```diff
@@ -219,3 +220,49 @@ func SupportsVsocks() bool {
 
 	return true
 }
+
+// refer to https://github.com/kubernetes/kubernetes/blob/v1.12.2/pkg/util/mount/mount_linux.go#L512
+func GetDevFormat(disk string) (string, error) {
```

Comment:
`blkid` is part of `util-linux` and is thus linux-specific. Please put it in `virtcontainers/utils/utils_linux.go` instead.

Reply:
Please also fix golint errors though as reported by travis.

Reply:
All right, These suggestions are good, I'll improve them.